apps/aeutils/test/data/epoch_full.yaml

```diff
@@ -94,6 +94,8 @@ mining:
     attempt_timeout: 3600000
     # Expected mine rate (milliseconds) between blocks. Used in governance.
     expected_mine_rate: 300000
+    # Public key of beneficiary account that will receive fees from mining on a node.
```

Comment:
This description sounds better than the one in the schema file, could you please update the schema description to match this one?